Web/Views/Viia/Accounts.cshtml

```diff
@@ -214,6 +218,16 @@
         
         return JsonConvert.SerializeObject(niceJsonObj, Formatting.Indented);
     }
+
+    Boolean hasNeverBeenSynced(DateTime? syncTime)
+    {
+        if(syncTime.Value.Year == 1970)
+        {
+            return true;
+        } else {
+            return false;
+        }
+    }
```

Comment:
I ended up writing long ass wall of text for a small thing, but hopefully it helps you learn a few small things about C#.

`?` at the end of the variable type (DateTime) means it's nullable, so doing `syncTime.Value.Year` can result in an exception.

```
bool HasNeverBeenSynced(DateTime? syncTime) => syncTime?.Year == 1970;
```
Should do the job.
To explain:
In C# naming convention:
 - methods start with an upper case
 - It's preferred to use aliases for basic types (e.g. `bool` instead of `Boolean` or `int` instead of `Int32`)
 - `=>` Makes a method into an expression, this is optional, code below works as well. In this case it's more of a style thing, making a method into one-liner.
```
bool HasNeverBeenSynced(DateTime? syncTime) {
    return syncTime?.Year == 1970;
}
```
 - using `?.` operator null checks it's value, if it's null, the whole operation's value pretty much becomes null.
So for example if `syncTime` is null, operation `syncTime?.Year == 1970`, gets compiled into something like `null == 1970` which will be false.

If syncTime being null is considered that account has never been synced, then the expression should be changed to 

```
bool HasNeverBeenSynced(DateTime? syncTime) => syncTime == null || syncTime.Year == 1970;
```
Note that null check operator `?.` is not needed, because we check if it's null in the expression and if it's null, it will immediately stop the expression since it evaluated to true and since next operator is `OR`, there's no need to check right side of the expression.

Reply:
Nice, ty for the comment/info - i'll fix it :)